keps/sig-release/create-conformance-doc.md

```diff
@@ -0,0 +1,87 @@
+---
+kep-number: draft-20190121
+title: Provide Conformance Document as part of the release docs for each Kubernetes minor release
+authors:
+  - "@brahmaroutu"
+owning-sig: sig-testing
```

Comment:
This is currently in the sig-release directory, but SIG Testing is listed as the owning SIG.
Who do we want to own this? This would need to be updated to that affect.

Reply:
sig-testing is not an appropriate owner here, I would suggest sig-architecture given that's where the conformance-definition subproject lives